client/src/components/application/NonlinearStepper/style.css

```diff
@@ -0,0 +1,198 @@
+.nonlinearSelect{
+  margin-bottom: 0.8rem;
+}
+.nonlinearSelect .MuiSelect-outlined.MuiSelect-outlined {
+  background-color: rgb(255, 255, 255);
+  display: flex;
+  align-items: center;
+  font-size: 14px;
+  margin-right: 28px;
+}
+
+
+.Extra_container_accounts {
+  background-color: rgb(245, 245, 245);
+  width: 100%;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding-bottom: 1%;
+}
+
+.Container_page__availability {
+  background-color: rgba(245, 245, 245, 0.411);
+  display: flex;
+  flex-direction: column;
+  padding-top: 2%;
+  padding-right: 10%;
+  width: 32rem;
+  z-index: 1;
+}
+
+.Container_page__availability .MuiStepper-horizontal{
+  background-color: transparent;
+}
+.Container_page__availability input {
+  height: 0.3rem;
+  background-color: rgb(255, 255, 255);
+  width: 30rem;
+  font-size: inherit;
+}
+
+.Container_page__availability .MuiSelect-selectMenu {
```

Comment:
Nice job putting `.Container_page__availability` in front of each of these selectors!